upi/openstack/bootstrap.yaml

```diff
@@ -20,6 +20,17 @@
       allowed_address_pairs:
       - ip_address: "{{ os_apiVIP }}"
 
+  - name: 'Create the bootstrap dualstack server port'
+    os_port:
+      name: "{{ os_port_bootstrap }}"
+      network: "{{ os_dualstack_network }}"
+      security_groups:
+      - "{{ os_sg_master }}"
+      allowed_address_pairs:
+      - ip_address: "{{ os_apiVIP }}"
+      - ip_address: "{{ os_apiVIP_v6 }}"
```

Comment:
should be `os_apiVIP6` to be consistent with https://github.com/openshift/installer/pull/7552/files#diff-5556986d20199572797cbcb320f2662d07506f6de617576f8ab15250e36b37efR10
similar with ingress below.